recipes/open62541/all/conanfile.py

```diff
@@ -31,7 +31,7 @@ class Open62541Conan(ConanFile):
         "shared": [True, False],
         "historize": [True, False, "Experimental"],
         "logging_level": ["Fatal", "Error", "Warning", "Info", "Debug", "Trace"],
-        "subscription": [True, False, "With Events"],
+        "subscription": [True, False, "With Events", "With Events, Alarms, Conditions"],
```

Comment:
Those are bad option values.
`With Events` has spaces and `With Events, Alarms, Conditions` has space+commas.
How about the following?
```python
        "subscription": [True, False, "events", "events+alarms+conditions"],
```
You can supper the old `With Events` by adding it to the list, and doing
```
def configure(self):
    if self.options.subscription == "With Events":
        self.output.warning("`{name}:subscription=With Events` is deprecated. Use `{name}:subscription=events` instead".format(name=self.name))  # Deprecated in 1.2.2
```
(and use `events`+`events+alarms+conditions` instead in `_configure_cmake`)

Reply:
Changed as suggested. Thanks!